datafusion/proto/Cargo.toml

```diff
@@ -28,13 +28,17 @@ keywords = ["arrow", "query", "sql"]
 edition = "2021"
 rust-version = "1.62"
 
+[package.metadata.docs.rs]
+features = [ "docsrs" ]  # https://github.com/rust-lang/docs.rs/issues/147
+
 [lib]
 name = "datafusion_proto"
 path = "src/lib.rs"
 
 [features]
 default = []
 json = ["pbjson", "pbjson-build", "serde", "serde_json"]
+docsrs = []
```

Comment:
Did I do this correctly? Does this declare a new feature but say it has no affect on dependencies?

Reply:
Yes, I think so